README.rst

```diff
@@ -37,8 +37,7 @@ environ_config: Configuration with env variables for Python.
   ...    lang = environ.var(name="LANG")  # It's possible to overwrite the names of variables.
   ...    db = environ.group(DB)
   ...    awesome = environ.bool_var()
-  >>> cfg = environ.to_config(
-  ...     AppConfig,
+  >>> cfg = AppConfig.from_environ(
```

Comment:
I don't think it's necessary to change the default here.

Reply:
Agreed. Done.